lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4970,6 +4970,9 @@ SwiftDeclConverter::importSwiftNewtype(const clang::TypedefNameDecl *decl,
       decl->getUnderlyingType(), ImportTypeKind::Value, isInSystemModule(dc),
       Bridgeability::None, OTK_None);
 
+  if (storedUnderlyingType.isNull())
```

Comment:
Nitpick: Type has an overload of `!` that can be used for this.

Reply:
No problem, I've used `isNull()` only because I've found it used elsewhere and I thought it is the preferred way to use, I'm fixing right away